pages/common/git-fame.md

```diff
@@ -0,0 +1,28 @@
+# git fame
+
+> Pretty-print Git repository contributions.
+> More information: <https://github.com/casperdcl/git-fame>.
+
+- Calculate contributions for the current Git repository:
+
+`git fame`
+
+- Exclude files/directories that match the specified regular expression:
+
+`git fame --excl "{{regular_expression}}"`
+
+- Calculate contributions made after the specified date:
+
+`git fame --since "{{3 weeks ago|2021-05-01}}"`
+
+- Display contributions in the specified format:
+
+`git fame --format {{pipe|md|markdown|yaml|yml|json|csv|tsv|tabulate}}`
```

Comment:
`pipe`, `md`, `markdown` and `tabulate` all look the same. Should we maybe only include `pipe`?

Reply:
Sure.  That's the default.
```suggestion
`git fame --format {{pipe|yaml|yml|json|csv|tsv}}`
```